src/directives/index.js

```diff
@@ -0,0 +1,344 @@
+      const innerHTML = el.innerHTML;
+      const position = binding.arg ? binding.arg : 'post';
+      const handlerElement = innerHTML => {
+        const value = binding.value !== null ?  i18nFnc(binding.value) : '';
+        if (position === 'pre') el.innerHTML =  `${value} ${innerHTML}`;
+        else if (position === 'post') el.innerHTML = `${innerHTML} ${value}`;
+      };
+      handlerElement(innerHTML);
+      return vm.$watch(() => ApplicationState.lng, () => handlerElement(innerHTML));
+    };
+
+    Vue.directive("disabled", vDisabled);
+    Vue.directive("checked", vChecked);
+    Vue.directive("selected-first", vSelectedFirst);
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-tooltip.js"):
+     */
+    Vue.directive('t-tooltip', {
+      bind(_el, binding) {
+        // handle automatic creation of tooltip
+        if (binding.modifiers.create) {
+          if (binding.arg){
+            _el.setAttribute('data-placement', binding.arg);
+            _el.classList.add(`skin-tooltip-${binding.arg}`);
+            _el.classList.add('skin-color');
+          }
+          const domelement = $(_el);
+          domelement.tooltip({
+            trigger : ApplicationState.ismobile ? 'click': 'hover',
+            html: true
+          });
+          // in case of mobile hide tooltip after click
+          ApplicationState.ismobile && domelement.on('shown.bs.tooltip', function(){
+            setTimeout(()=>$(this).tooltip('hide'), 600);
+          });
+        }
+
+        const unique_v_t_tooltip_attr = createDirectiveObj({
+          el:_el,
+          attr: 'g3w-v-t-tooltip-id'
+        });
+        const i18Fnc = binding.arg;
+        directives[unique_v_t_tooltip_attr].modifiers = binding.modifiers;
+        const handler = ({el=_el}={}) =>{
+          const current_tooltip = el.getAttribute('current-tooltip');
+          const unique_v_t_tooltip_attr =  el.getAttribute('g3w-v-t-tooltip-id');
+          const value = current_tooltip !== null ? current_tooltip:  binding.value;
+          const title = directives[unique_v_t_tooltip_attr].modifiers.text  ? value : (i18Fnc === 'plugin') ? tPlugin(value) : t(value);
+          el.setAttribute('data-original-title', title);
+        };
+        handler();
+        directives[unique_v_t_tooltip_attr].handler = handler;
+        setUnwatch({
+          id:unique_v_t_tooltip_attr,
+          unwatch: vm.$watch(() => ApplicationState.lng, handler)
+        })
+      },
+      componentUpdated(el, oldVnode){
+        runHandlerOnUpdate({
+          el,
+          attrId: 'g3w-v-t-tooltip-id',
+          attr:'current-tooltip',
+          oldValue: oldVnode.oldValue
+        });
+      },
+      unbind(el){
+        $(el).tooltip('hide');
+        unbindWatch({
+          attr:'g3w-v-t-tooltip-id',
+          el
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-html.js"):
```

Comment:
@volterra79 does it seem feasible to you?

Reply:
Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00